zer0-os/zOS
# feat(conversation-list-item/matrix-client): handle message previews for reaction events on initial load

### What does this do?
- handles message previews for reaction events

### Why are we making this change?
- improved ui / ux
- ensure we display content for reactions in conversation list item prevew

### How do I test this?
- run tests as usual
- run UI > navigate to social channel > add reaction & check preview.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


<img width="280" alt="Screenshot 2024-10-08 at 11 25 29" src="https://github.com/user-attachments/assets/f15b7201-9589-4bfb-b0ce-d035815640d7">
